Syntax.in

```diff
@@ -0,0 +1,29 @@
+The words - predefined tokens are specified between " and ":  
+
+<program> ::= "##GO_PROGRAM" <statement-list> "##STOP_PROGRAM"
+    <statement-list> ::= <statement> | <statement> ";" <statement-list>
+        <statement> ::= <declaration-statement> | <simple-statement> | <compound-statement>
+
+            <declaration-statement> ::= <simple-declaration> | <array-declaration>
+                <simple-declaration> ::= "def" <type> <identifier> | "def" <type> <identifier> <initial-value>
+                <array-declaration> ::= "defArray" <type> <identifier> | "defArray" <type> <identifier> <initial-array-value>
+
+                <type> ::= "integer" | "char" | "string"
+                <initial-array-value> ::= "[" <list-of-initial-values> "]"
+                <list-of-initial-values> ::= <initial-value> | <initial-value><list-of-initial-values>

```

Comment:
How you do separate values from the list? Maybe put a comma or semicolumn or something like that between them.